frontend/benefit/handler/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@frontend/bf-handler",
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "dev": "node start.js",
+    "build": "next build",
+    "start": "NODE_ENV=production node start.js",
+    "lint": "eslint --ext js,ts,tsx src",
+    "pre-commit": "lint-staged -c ../../.lintstagedrc.js",
+    "test": "jest --passWithNoTests",
+    "test:staged": "yarn test --watchAll=false --findRelatedTests",
+    "test:coverage": "yarn test --verbose --coverage",
+    "browser-test": "testcafe \"chrome --window-size='1249,720'\" testcafe.config.js browser-tests/",
+    "browser-test:ci": "testcafe \"chrome:headless --disable-gpu --window-size='1249,720'\" --screenshots takeOnFails=true,path=report,fullPage=true --video report --reporter spec,html:report/index.html testcafe.config.js browser-tests/"
+  },
+  "dependencies": {
+    "@frontend/shared": "*",
+    "@types/next": "^9.0.0",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.6",
+    "@types/react-input-mask": "2.0.4",
+    "axios": "^0.21.1",
+    "babel-plugin-styled-components": "^1.13.2",
+    "camelcase-keys": "^7.0.0",
+    "dotenv": "^9.0.0",
+    "esm": "^3.2.25",
+    "formik": "^2.2.9",
+    "graphql": "^15.5.0",
+    "hds-react": "^1.3.1",
+    "lodash": "^4.17.21",
+    "next": "^11.1.0",
+    "next-compose-plugins": "^2.2.0",
+    "next-i18next": "^8.4.0",
+    "next-plugin-custom-babel-config": "^1.0.5",
+    "next-transpile-modules": "^8.0.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-i18next": "^11.11.0",
+    "react-input-mask": "^2.0.4",
+    "react-loading-skeleton": "^2.2.0",
+    "react-query": "^3.17.0",
+    "react-toastify": "^8.0.2",
+    "rexreplace": "^7.0.3",
```

Comment:
I saw this package listed in other services, but are we using it?

Reply:
fixed